src/el/el_client_context.star

```diff
@@ -7,7 +7,8 @@ def new_el_client_context(
 	ws_port_num,
 	engine_rpc_port_num,
 	jwt_secret,
-	service_name = ""):
+	service_name = "",
+	electra_fork_epoch = None):
```

Comment:
I don't think you need this here in el_client_context!

Reply:
you are right